.github/workflows/build.yml

```
@@ -310,94 +310,30 @@ jobs:
-
-    - name: Create GitHub Release
-      uses: softprops/action-gh-release@v2
-      with:
-        files: |
-          ./artifacts/**/*.hap
-          ./artifacts/**/*.app
-        draft: false
-        prerelease: ${{ contains(github.ref, 'beta') || contains(github.ref, 'alpha') || contains(github.ref, 'rc') }}
-        generate_release_notes: true
+  # Release 由本地手动构建签名后上传
         fail_on_unmatched_files: false
```

actionlint (1.7.11)

[error] input "fail_on_unmatched_files" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)